packages/m/ManUp.js.json

```
@@ -0,0 +1,23 @@
+{
+	"name": "ManUp.js",
+	"version": "",
+	"description": "ManUp.js is a polyfill to support the Manifest for Web Apps",
+	"filename": "manup.min.js",
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/boyofgreen/ManUp.js.git"
+	},
+	"keywords": ["boyofgreen", "polyfill", "manup", "manupjs"],
```

```suggestion
	"keywords": [
	  "boyofgreen",
	  "polyfill",
	  "manup",
	  "manupjs"
	],
```
Formatting consistency :)